nzediegwu1/blog-connect-web
# #161525740 Setup continuous deployment to Google Kubernates

#### What does this PR do?
Setup continuous deployment from circleci to google kubernates engine

#### Description of Task to be completed?
- Define circleci deployment config
- Define kubernates deployment workflow including setting the number of pods, port to expose to public

#### How should this be manually tested?
GIVEN application URLs
WHEN user navigates to provided URLs in the browser
THEN user should access the live application

GIVEN a developer raises a `Pull Request`
WHEN the PR is merged/ commit pushed to develop branch
THEN the changes should be deployed to develop branch, and can be view via URL

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161525740](https://www.pivotaltracker.com/story/show/161525740)

#### Screenshots (If applicable)
N/A
